extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -436,6 +441,15 @@ const actions = (state, action, data) => {
         ...state,
         [StateProperty.VIEWER_CUSTOM_CONTROLS]: data,
       });
+    case Action.TOGGLE_SUBSCRIPTION_DIALOG_IS_VISIBLE:
+      // Don't change the state if the new state is equal to the current state.
```

Comment:
By definition, it's not a change if they're equal. This if() is not needed

Later on in the execution, the new state will be compared with the old state. If there was no change, callbacks won't be executed.

Reply:
Removed. Thanks for the info.